components/zeus_ui/fn_addZeusActions.sqf

```diff
@@ -0,0 +1,151 @@
+/* --------------------------------------------------------------------------------------------------------------------
+	Author:		Joecuronium
+	Description:
+		Adds actions to ZEUS ACE-interact that allows Zeus to choose certain behaviours in relation to ACRE and self-teleport.
+	Arguments:
+		(nothing)
+	Returns:
+		(nothing)
+-------------------------------------------------------------------------------------------------------------------- */
+
+
+#include "macros.hpp"
+
+CLIENT_ONLY;
+
+if(!(player getVariable ["f_var_isZeus",false])) exitWith {}; //Fallback in case the script execution is run on the wrong machine
+
+
+//ACRE actions
+private _script =
```

Comment:
Descriptive variable names makes it easier to understand what the code does when you read through it later on.
```suggestion
private _speakThroughZeusCamera =
```